edk2toolext/environment/plugin_manager.py

```diff
@@ -10,7 +10,6 @@
 import os
 import imp
 import logging
-from edk2toolext.environment import shell_environment
```

Comment:
By moving this to ci_build, is the implication that build_plugins can no longer be skipped on a per-platform basis?